components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/inbound/websocket/request/GraphQLRequestProcessor.java

```diff
@@ -108,6 +111,20 @@ public InboundProcessorResponseDTO handleRequest(int msgSize, String msgText,
                                 // set resource name of subscription operation for analytics event publishing
                                 WebSocketUtils.setApiPropertyToChannel(inboundMessageContext.getCtx(),
                                         APIConstants.API_ELECTED_RESOURCE, subscriptionOperation);
+                                WebSocketUtils.setApiPropertyToChannel(inboundMessageContext.getCtx(),
+                                        APIConstants.GRAPHQL_SCHEMA, inboundMessageContext.getGraphQLSchemaDTO().getGraphQLSchema());
+                                WebSocketUtils.setApiPropertyToChannel(inboundMessageContext.getCtx(),
+                                        APIConstants.TYPE_DEFINITION, inboundMessageContext.getGraphQLSchemaDTO().getTypeDefinitionRegistry());
+
+                                try {
+                                    String accessControlInfo = getGraphQLAccessControlInfo(inboundMessageContext.getGraphQLSchemaDTO()
```

Comment:
Seems max length is exceeding.

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/12441/commits/226c49a261240dc5305f96d69aadb6f86edfae84